.github/workflows/tests.yml

```diff
@@ -13,14 +13,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, '3.10']
+        python-version: [3.7, 3.8, 3.9, '3.10','3.13-dev']
```

Comment:
Could you add 3.11 & 3.12 here as well? AFAIK this simply hasn't been updated for some time.

We might also consider dropping some older versions, but that can be done separately.

Reply:
I would drop 3.7, but leave 3.8 since it only went eol last month